contracts/Quest.sol

```diff
@@ -254,7 +245,11 @@ contract Quest is ReentrancyGuardUpgradeable, PausableUpgradeable, Ownable, IQue
     /*//////////////////////////////////////////////////////////////
                              INTERNAL VIEW
     //////////////////////////////////////////////////////////////*/
-    function _receiptRedeemers() internal view returns (uint256) {
+    function _redeemedTokens() internal view returns (uint256) {
         return questFactoryContract.getNumberMinted(questId);
     }
+
+    function _claimFee() internal view returns (uint256) {
+        return questFactoryContract.mintFee();
```

Comment:
We could potentially cache this value or pass it through in the `claimFromFactory` call to prevent the callback. Obviously caching it is functionally different.